zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -115,8 +132,9 @@ const BIT_LEN_LAST_LIMB: usize = 256 - (NUMBER_OF_LIMBS - 1) * BIT_LEN_LIMB;
 
 /// SignVerify Configuration
 #[derive(Debug, Clone)]
-pub(crate) struct SignVerifyConfig {
+pub(crate) struct SignVerifyConfig<F: Field> {
     // ECDSA
+    ecdsa_config: FpChip<F>,
     main_gate_config: MainGateConfig,
     range_config: RangeConfig,
```

Comment:
We need to remove the configs used by halo2wrong.

Reply:
removed range_config etc. still need to use main_gate_config for keccak lookup